tailwind.config.js

```diff
@@ -16,7 +23,7 @@ module.exports = {
     },
     fontFamily: {
       body: ["Outfit"]
-    }
```

Comment:
We already changed the body font to the `outfit` @DevRamona can help you with that